CHANGELOG.md

```diff
@@ -78,6 +78,11 @@ Deprecated names
 New modules
 -----------
 
+* Pointwise algebraic structures and bundles:
+  ```
+  Algebra.Construct.Pointwise
```

Comment:
The eternal naming question: I had no idea what this was until I read further down. `PointwiseLift` ? `FunctionsOver` (that's bad, I know, but something with 'Function' in the name is not unreasonable)?

Reply:
Revised description...